src/LiquidityHelperContract.sol

```diff
@@ -2,11 +2,14 @@
 
 pragma solidity ^0.8.20;
 
-import {ILBPair, LiquidityHelper} from "./periphery/LiquidityHelper.sol";
+import {ILBPair} from "joe-v2/interfaces/ILBPair.sol";
+
+import {LiquidityHelper} from "./periphery/LiquidityHelper.sol";
+import {NonEmptyBinHelper} from "./periphery/NonEmptyBinHelper.sol";
 
 /**
- * @title Liquidity Book periphery contract for Liquidity and Fees Amounts
- * @notice Periphery contract to help compute liquidity and fees amounts from amounts and ids.
+ * @title Liquidity Book periphery contract for Liquidity, Fees Amounts and bin fetching.
+ * @notice Periphery contract to help compute liquidity, fees amounts from amounts and ids and fetch bins.
```

Comment:
maybe worth adding, that this contract should be called only off-chain, based on quoter experience

Reply:
Agree lol